app/src/main/java/com/willowtree/vocable/CategoriesUseCase.kt

```diff
@@ -49,10 +46,9 @@ class CategoriesUseCase(
     }
 
     override suspend fun addCategory(categoryName: String, sortOrder: Int) {
```

Comment:
Separate discussions from your code but I'm curious on your thoughts on these sort of mono use cases.

Reply:
Meaning, use cases that can do more than one thing? I don't have super strong feelings, and I know a lot of folks go with the one-operation use cases, but I find that doesn't buy you a whole lot for the extra verbosity. What are your thoughts?

Reply:
It doesn't buy you much for sure. It feel like its conflating two patterns but doesn't feel too wrong otherwise.